.github/workflows/cirque.yaml

```diff
@@ -77,6 +77,7 @@ jobs:
             - name: Bootstrap
               timeout-minutes: 15
               run: |
+                  git config --global --add safe.directory /home/runner/work/connectedhomeip/connectedhomeip
```

Comment:
I am unsure if this is the right place and also the path is hard-coded, maybe using `pwd` is better. 
In any case, this still fails so I am guessing something in run.sh creates some separate environment, so that needs investigation.

Reply:
Updated